test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -138,6 +138,21 @@ var _ = Describe("[performance] Checking IRQBalance settings", Ordered, func() {
 				return nil
 			}
 
+			defer func() { // return initial configuration
+				By("reverting the profile into its initial state")
+				spec, err := json.Marshal(initialProfile.Spec)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(testclient.Client.Patch(context.TODO(), profile,
+					client.RawPatch(
+						types.JSONPatchType,
+						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
+					),
+				)).ToNot(HaveOccurred())
+				// in the initial profile `GloballyDisableIrqLoadBalancing` value should be false,
+				// so we expect the banned cpu set to be empty
```

Comment:
It is not needed, but still beneficial, to annotate the `Eventually` expectation to explicitly convey that if this fails, it's because the profile failed to revert

Reply:
Why do you think the wait is not necessary?

Reply:
The wait is necessary, I just think is better to be extra explicit about what the failure means.

Reply:
oh I see, I added the comment just to explain why we're passing `false` argument to the comment and not why we add `Eventually`